Search.IndexService/Internal/ParsedHtml.cs

```diff
@@ -1,8 +1,14 @@
-﻿namespace Search.IndexService.Internal
+﻿using System.Collections.Generic;
+
+namespace Search.IndexService.Internal
 {
     internal class ParsedHtml
     {
         public string Title { get; set; }
+
         public string Text { get; set; }
+
+        public List<string> Links;
```

Comment:
Лучше всё-таки свойством сделать, а не полем
```suggestion
        public List<string> Links { get; set; };
```